google/data_source_google_folder.go

```diff
@@ -0,0 +1,59 @@
+package google
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	resourceManagerV2Beta1 "google.golang.org/api/cloudresourcemanager/v2beta1"
+	"google.golang.org/api/googleapi"
+)
+
+func dataSourceGoogleFolder() *schema.Resource {
```

Comment:
Could you rename this method to match the new resource name along with the read method